buildconfig/CMake/MSVCSetup.cmake

```diff
@@ -88,12 +88,18 @@ set ( CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin )
 ###########################################################################
 set ( WINDOWS_BUILDCONFIG ${PROJECT_SOURCE_DIR}/buildconfig/windows )
 configure_file ( ${WINDOWS_BUILDCONFIG}/buildenv.bat.in ${PROJECT_BINARY_DIR}/buildenv.bat @ONLY )
-configure_file ( ${WINDOWS_BUILDCONFIG}/command-prompt.bat ${PROJECT_BINARY_DIR}/command-prompt.bat @ONLY )
 
-get_filename_component ( MSVC_IDE_LOCATION "${CMAKE_CXX_COMPILER}" DIRECTORY )
-get_filename_component ( MSVC_IDE_LOCATION "${MSVC_IDE_LOCATION}/../../../../../../.." ABSOLUTE )
-set ( MSVC_IDE_LOCATION "${MSVC_IDE_LOCATION}/Common7/IDE")
+if ( MSVC_VERSION EQUAL "1911" )
```

Comment:
Should we switch this around and check `< 1911` and do the VS140 stuff else do the VS2017 and beyond steps? In the future I presume newer versions will follow the VS2017 convention and then we won't have to update this code.

Reply:
Good point